low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -105,680 +38,21 @@ def get_health_info(self, rpath):
-        logger.debug(self.log.svc_log(
-            f"Platform Sensor Health Data:{response}"))
-        return response
-
-    def get_mem_info(self):
-        """Collect & return system memory info in specific format."""
-        default_mem_usage_threshold = int(Conf.get(SSPL_CONF,
-            "NODEDATAMSGHANDLER>host_memory_usage_threshold",
-            80))
-        data = []
-        status = "OK"
-        description = "Host memory is in good health."
-        self.mem_info = dict(psutil.virtual_memory()._asdict())
-        curr_mem_usage_threshold = int(self.mem_info['percent'])
-        if curr_mem_usage_threshold > int(default_mem_usage_threshold):
-            status = "Overloaded"
-            description = (f"Current host memory usage is {curr_mem_usage_threshold},"
-                           f"beyond configured threshold of {default_mem_usage_threshold}.")
-
-        memory_dict = self.prepare_mem_json(status,
-                                            description)
-        data.append(memory_dict)
-        logger.debug(self.log.svc_log(
-            f"Memory Health Data:{data}"))
-        return data
-
-    def prepare_mem_json(self, status, description):
-        """Update and return memory information dict."""
-        total_memory = {}
-        for key, value in self.mem_info.items():
-            if key == 'percent':
-                total_memory['percent'] = str(self.mem_info['percent']) + '%'
-            else:
-                total_memory[key] = str(self.mem_info[key] >> 20) + 'MB'
-        uid = "main_memory"
-        specifics = [
-                    {
-                        "total": total_memory['total'],
-                        "available": total_memory['available'],
-                        "percent": total_memory['percent'],
-                        "used": total_memory['used'],
-                        "free": total_memory['free'],
-                        "active": total_memory['active'],
-                        "inactive": total_memory['inactive'],
-                        "buffers": total_memory['buffers'],
-                        "cached": total_memory['cached'],
-                        "shared": total_memory['shared'],
-                        "slab": total_memory['slab']
-                    }
-                ]
-        memory_dict = self.get_health_template(uid, is_fru=False)
-        self.set_health_data(
-            memory_dict, status=status, description=description,
-            specifics=specifics)
-        return memory_dict
-
-    def get_memory_overall_usage(self):
-        """Returns Memory overall usage."""
-        overall_usage = None
-        mem_info = self.get_mem_info()
-        if mem_info[0].get("health"):
-            overall_usage = mem_info[0]["health"]["specifics"]
-        else:
-            logger.error(
-                self.log.svc_log("Failed to get memory overall usage"))
-        return overall_usage
+        Fetch manifest information for given rpath.
```

Comment:
Fetch manifest information for given resource path

Reply:
Done